opendebates/tests/test_context_processors.py

```diff
@@ -22,6 +22,7 @@ class ThemeTests(TestCase):
     def setUp(self):
         self.idea = SubmissionFactory()
 
+    @override_settings(SITE_THEME={'HASHTAG': 'TestHastag'})
     def test_email_url(self):
         settings.SITE_THEME['HASHTAG'] = 'TestHashtag'
```

Comment:
This is now harmless but unnecessary, right?  (If the typo in the `@override_settings` value is fixed)